### DISCUSSION TITLE
Bundle size / mapbox-gl included multiple times

### DISCUSSION THREAD

**Discussion by EvanAgee** [uber/react-map-gl#1397] [Q&A]
I have two components that import components from `react-map-gl`. When analyzing the bundle I'm seeing that `mapbox-gl` is being imported twice. Any way to fix this? It's doubling my bundle size.

**The imports:**
![image](https://user-images.githubusercontent.com/1692674/110804211-9212ec00-8245-11eb-8af7-50ea0e0091e8.png)

**The bundle analysis:**
![image](https://user-images.githubusercontent.com/1692674/110803977-5b3cd600-8245-11eb-8d8a-5486c1823fb3.png)

**Comment by Pessimistress** [comment]
You likely have multiple copies of mapbox-gl installed. You can check it with `npm ls mapbox-gl` or `yarn list mapbox-gl`.
